code-injection/Kaleidoscope-Inc_test-express-api-bom/app.js

```
@@ -0,0 +1,43 @@
+  const user = new User(req.body);
+  
+  user.save()
+    .then(savedUser => res.status(201).json(savedUser))
+    .catch(err => res.status(400).json({ error: err.message }));
+});
+
+// VULNERABILITY: Using eval with user input
+app.get('/search', (req, res) => {
+  const query = req.query.q || '';
+  try {
+    const result = eval('(' + query + ')');
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)